### PR TITLE
[DUOS-2829][risk=no] Use new Sam ToS endpoints

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2829

### Summary
Update the current POST/DELETE consent APIs to point to the new PUT ToS endpoints in Sam.
No changes are necessary in the UI since the UI isn't making use of the responses to the current Consent APIs.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/test/java/org/broadinstitute/consent/http/resources/SamResourceTest.java [hunk 1]
```diff
@@ -47,16 +48,15 @@ public class SamResourceTest {
   private SamResource resource;
 
   @BeforeEach
-  public void setUp() {
-    openMocks(this);
+  void setUp() {
   }
```
Comment (fboulnois) [inline src/test/java/org/broadinstitute/consent/http/resources/SamResourceTest.java]: Is this needed if no setup is required?

**Review by fboulnois — APPROVED** [review]
Overall looks good, one small question:

File: src/test/java/org/broadinstitute/consent/http/resources/SamResourceTest.java (hunk 1 above)
Comment (rushtong) [reply src/test/java/org/broadinstitute/consent/http/resources/SamResourceTest.java]: No, it is not, thank you!

**Review by aarohinadkarni — APPROVED** [review]
👍

**Pull request merged**